### DISCUSSION TITLE
aboutme

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#12] [Announcements]
# aboutme

https://nonzerosum.games/aboutme.html

**Comment by coppinger** [comment]
Hey James, love the site. 3D tetris is quite the mindblower—I'm glad you managed to get some sleep in the end, though :)

Quick question: who is the ZeFrank that you attribute the quote to at the bottom of your about me page?

I did some Googl'ing, and wasn't sure if it was the internet figure who hosts the ZeFrank show (which I'd never heard of until now), or someone else. It's a really great quote.

Many thanks,

Charlie

**Reply by jamesstephenbrown** [reply]
Hey Charlie, thanks for the feedback, 3D Tetris was fun to nut out.

If you haven't met ZeFrank yet, you are in for a treat - yes, he's the internet figure who hosts "The Show With ZeFrank" from 2007 which has been archived [here](https://www.youtube.com/@theshowwithzefrank06/videos?view=0&sort=dd&shelf_id=0) and is a weekly comedy show, that dealt with current events and various creative projects and "*A* Show With ZeFrank" [here](https://www.youtube.com/watch?v=RYlCVwxoL_g&list=PLkXVFxHhJSIbC09cYmm2d4KR07rw6xne7) which came a few years later and is more about creative process - this is where the quote is from. He's currently doing a series on [True Facts](https://www.youtube.com/playlist?list=PLOHbM4GGWADc5bZgvbivvttAuWGow6h05) about animals which is pretty funny. He also make a whole load of stuff in the early 2000s using Flash, meaning they are now all unfortunately defunct.

He's been a major influence for me being involved in the ongoing conversation that is the internet. My favourite quote of his is actually "The things that make us feel most alone have the greatest power to connect us" I'm planning a blog on this topic soon.

**Reply by coppinger** [reply]
Got it, thanks for clarifying James. I'm excited to tune in to some ZeFrank stuff, he seems like a really cool (and entertaining) character.

I'm not sure if you would have heard of it before, but there's an old comedy show from UK radio called 'The Ricky Gervais Show', co-hosted with Karl Pilkington and Steven Merchant (somewhat before the three of them were household names (at least in the UK)).

> Ricky Gervais and Steve Merchant returned to XFM, the alternative London-based radio station in September 2001 after the first series of The Office had been broadcast. Due to the phenomenal success of the show, Ricky was important enough to now be given his own producer, one Karl Pilkington. Although Karl was hired to just "press the buttons", Ricky and Steve got him involved more and more with the show over the subsequent weeks and soon became fascinated with his personal life, unconventional childhood and ridiculous stories. By the end of the first season Karl had become a crucial part of the show's success.

You can find the archive of episodes [here on the XFM website](https://www.therickygervaisshow.com/xfm-vault)—a comedy show trade, if you will :)

I absolutely love your focus on positive sum games—it's something I've been thinking about and trying to base my life around for the last few years. There are a lot of games played at scale within society that could be quite easily turned from zero-sum into positive-sum, I suspect. I hope to contribute to that endeavor in a small but meaningful way within my lifetime.

Is there an email address/contact form I can reach you via? I'd love to share a project I'm working on with you, but I don't want to derail your comment section!

Best,

Charlie

**Reply by jamesstephenbrown** [reply]
Hey Charlie,
Ha, yes I have listened to the entire catalogue of The Ricky Gervais Show, I can see we are clearly on the same wavelength. Thanks so much for your kind words about the site, I really began the site to find like-minded people who were interested in these sorts of ideas, so I'm very glad to have met you. Feel free to email me at jamesstephenbrown@gmail.com and please don't worry at all about derailing a comments thread here, the more comments the merrier.
James :)
